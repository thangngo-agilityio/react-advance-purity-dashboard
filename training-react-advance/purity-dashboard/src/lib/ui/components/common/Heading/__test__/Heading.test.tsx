import { render } from "@testing-library/react"

// Component
import { Heading } from "../.."

describe('Heading test cases', () => {
  it('should render correctly', () => {
    const { container } = render(
      <Heading title={"Test"}>
      </Heading>
    );
    expect(container).toMatchSnapshot();
  })
});
