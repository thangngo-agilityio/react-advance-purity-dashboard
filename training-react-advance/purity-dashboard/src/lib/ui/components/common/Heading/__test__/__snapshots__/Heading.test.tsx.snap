// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heading test cases should render correctly 1`] = `
<div>
  <h2
    class="chakra-heading css-0"
  >
    Test
  </h2>
</div>
`;
